src/pages/import/components/ImportTimetable.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+  useToast,
+  VStack,
+  Text,
+  Alert,
+  AlertIcon,
+  useMediaQuery,
+} from '@chakra-ui/react';
+import { BsPlusCircleFill } from 'react-icons/bs';
+
+import { Timetable } from 'lib/fetchers';
+import { SavedCourse, useSavedCourses } from 'lib/hooks/useSavedCourses';
+
+import { SelectedCoursesCardList } from 'pages/scheduler/components/share/SelectedCoursesCardList';
+
+export function ImportTimetable({ loading, data }: { loading: boolean; data: Timetable }): JSX.Element {
+  const { importCourses } = useSavedCourses();
+  const toast = useToast();
+  const [isSmallScreen] = useMediaQuery('(min-width:680px)');
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const filteredCoursesList: SavedCourse[] = [];
+  const [filteredCourses, setFilteredCourses] = useState(filteredCoursesList);
+  const [timetableTerm, setTimetableTerm] = useState('');
+
+  const handleImport = () => {
+    importCourses(data.courses, data.term);
+    onClose();
+    toast({
+      title: 'Timetable Imported!',
```

Comment:
```suggestion
      title: 'Timetable imported!',
```
Just for consistency with our other toast usage we use sentence case